tests/e2e/Services/Databases/DatabasesBase.php

```diff
@@ -297,6 +297,174 @@ public function testCreateAttributes(array $data): array
+        $this->assertEquals($databaseId, $document['body']['$databaseId']);
+        $this->assertEquals($document['body']['1'], 'one');
+        $this->assertEquals($document['body']['200'], 'two hundred');
+        $this->assertEquals($document['body']['1000000'], 'one million');
+        $this->assertIsArray($document['body']['$permissions']);
+        $this->assertCount(3, $document['body']['$permissions']);
+
+        // Update document
+        $document = $this->client->call(Client::METHOD_PATCH, '/databases/' . $databaseId . '/collections/' . $numericKeyId . '/documents/' . $id, array_merge([
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+        ], $this->getHeaders()), [
+            'data' => [
+                '1' => 'one updated',
+                '200' => 'two hundred updated',
+                '1000000' => 'one million updated'
+            ],
+            'permissions' => [
+                Permission::read(Role::users()),
+                Permission::update(Role::users()),
+                Permission::delete(Role::users()),
+            ],
+        ]);
+
+        $this->assertEquals(200, $document['headers']['status-code']);
+        $this->assertEquals($document['body']['$id'], $id);
+        $this->assertEquals($numericKeyId, $document['body']['$collectionId']);
+        $this->assertArrayNotHasKey('$collection', $document['body']);
+        $this->assertEquals($databaseId, $document['body']['$databaseId']);
+        $this->assertEquals($document['body']['1'], 'one updated');
+        $this->assertEquals($document['body']['200'], 'two hundred updated');
+        $this->assertEquals($document['body']['1000000'], 'one million updated');
+        $this->assertContains(Permission::read(Role::users()), $document['body']['$permissions']);
+        $this->assertContains(Permission::update(Role::users()), $document['body']['$permissions']);
+        $this->assertContains(Permission::delete(Role::users()), $document['body']['$permissions']);
+
+        sleep(2);
+
+        $document = $this->client->call(Client::METHOD_GET, '/databases/' . $databaseId . '/collections/' . $numericKeyId . '/documents/' . $id, array_merge([
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+        ], $this->getHeaders()));
+
+        $id = $document['body']['$id'];
+
+        $this->assertEquals(200, $document['headers']['status-code']);
+        $this->assertEquals($numericKeyId, $document['body']['$collectionId']);
+        $this->assertArrayNotHasKey('$collection', $document['body']);
+        $this->assertEquals($databaseId, $document['body']['$databaseId']);
+        $this->assertEquals($document['body']['1'], 'one updated');
+        $this->assertEquals($document['body']['200'], 'two hundred updated');
+        $this->assertEquals($document['body']['1000000'], 'one million updated');
```

Comment:
Let's remove these assertions to make the test easier to read, we only need to check that the update was successful